blob_server/models.py

```diff
@@ -35,15 +34,33 @@ class BlogPost(Base):
         secondary='post_tag_links'
     )
 
+    @property
+    def serializable(self) -> dict:
+        tags = [{'id': t.id, 'name': t.name} for t in self.tags]
+        return {
+            'id': self.id,
+            'created': self.created,
+            'content': self.content,
+            'tags': tags,
```

Comment:
FYI, you can define a list using a comprehension inline with the dictionary, like:

```python
return {
   'tags': [{'id': t.id, 'name': t.name} for t in self.tags]
}
```